.github/workflows/stale-issues-workflow.yml

```diff
@@ -1,10 +1,13 @@
+---
```

Comment:
See my comment below about yamllint. If you execute the linter through `bitrise run check`, then it will ignore the yml files in the `.github` folder.

Reply:
- [ ] Revert changes to the stale-issues-workflow.yml file

Reply:
@hisaac I still see some changes in this file

Reply:
Oops, my bad. I reverted one commit to that file, but didn't realize there were other changes. Will fix.

- [ ] _Actually_ revert all the changes to the `stale-issues-workflow.yml` file

Reply:
OK, I reverted all the changes now 👍